src/extension.ts

```diff
@@ -184,18 +207,37 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
         synchronize: {
             // Synchronize the setting section 'psalm' to the server (TODO: server side support)
             configurationSection: 'psalm',
-            fileEvents: vscode.workspace.createFileSystemWatcher('**/' + psalmConfigPath)
+            fileEvents: [
+                vscode.workspace.createFileSystemWatcher('**/' + psalmConfigPath),
+                // this is for when files get changed outside of vscode
+                vscode.workspace.createFileSystemWatcher('**/*.php'),

```

Comment:
I added this for when files may get edited outside of vscode for whatever reason (package updates, project scripts, etc.)